vue.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  extends: [
+    "airbnb-typescript",
+    "eslint-config-prettier",
+    "plugin:vue/vue3-strongly-recommended",
+    "./base.js",
+  ],
+  env: {
+    browser: true,
+  },
+  parser: "vue-eslint-parser",
+  parserOptions: {
+    ecmaFeatures: {
+      globalReturn: false,
+      impliedStrict: false,
+      jsx: false
+    },
+    parser: "@typescript-eslint/parser"
+  },
+  rules: {
+    "vue/html-closing-bracket-newline": "off",
+    "vue/html-closing-bracket-spacing": "off",
```

Comment:
Why are these rules being overridden?

Reply:
Prettier should take care of these, no?

Reply:
OK, I think we can remove them. Thanks for pointing out that